elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchCalendarBridge.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.impl;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import org.apache.lucene.document.Document;
+import org.hibernate.search.annotations.Resolution;
+import org.hibernate.search.backend.elasticsearch.util.impl.ElasticsearchDateHelper;
+import org.hibernate.search.bridge.LuceneOptions;
+
+/**
+ * Bridge a {@code java.util.Calendar} to a {@code String} using the ISO 8601 standard which is the default date format
+ * of Elasticsearch.
+ * UTC is used as time zone.
```

Comment:
Did you consider to store the TZ offset instead (it's my understanding that ES supports formats with a TZ offset)? That way the calendar's TZ info can e.g. be shown when projecting the source document (or just a single field). That'd more be what I'd expect as a user.